components/script/dom/globalscope.rs

```diff
@@ -3498,6 +3498,48 @@ impl GlobalScope {
             self.resolved_module_set.borrow_mut().insert(record);
         }
     }
+
+    /// <https://html.spec.whatwg.org/multipage/#run-steps-after-a-timeout>
+    pub(crate) fn run_steps_after_a_timeout<F>(
```

Comment:
I don't see any usages of this method. Why isn't it marked as unused and does Clippy fail on it?

Reply:
you can check #40032